crates/ruff/src/settings/defaults.rs

```diff
@@ -28,6 +28,8 @@ pub const TARGET_VERSION: PythonVersion = PythonVersion::Py310;
 
 pub const LINE_LENGTH: usize = 88;
 
+pub const TAB_SIZE: u8 = 4;
```

Comment:
Nit: It could be worth introducing a `TabSize` newtype wrapper so that we can change the internal representation in a single place if we ever want to.

Reply:
Done!